### ISSUE TITLE
Ошибка при загрузке edf файла 

### ISSUE THREAD

**Issue by logogriph2** [dmalt/cognigraph#13]
При попытке загрузить edf, для которого была посчитана модель DF_2018-03-02_11-34-38-fwd.fif возникла ошибка, связанная, насколько я понимаю, с числом каналов. Если я правильно понял, канал с метками был принят за канал с данными
![fastovets_error](https://user-images.githubusercontent.com/24354709/46089771-e7fb9280-c1b7-11e8-8124-f6d7dd41f542.jpg)

**Comment by dmalt** [comment]
Вы использовали вот этот [файл](https://drive.google.com/open?id=12cZzF4-dyFTx7R_WAKutvhYzCCGdpnOm) с gdrive? Если нет, можете прислать ваши данные, я у себя запущу.

**Comment by logogriph2** [comment]
Да, я использовал этот же файл.

**Comment by logogriph2** [comment]
Проблема все ещё актуальна.

**Comment by dmalt** [comment]
Поправил. Попутно обнаружил баг в нормировке цветовой шкалы для метода MCE. Его тоже починил.

**Comment by logogriph2** [comment]
Скачал последнюю версию. Программа зависла при расчете отображения источников.

![systemtest_mce_zavis](https://user-images.githubusercontent.com/24354709/46472743-d41ee480-c7e6-11e8-9f1c-7307ac6bd1f4.png)
![systemtest_mce_zavis_log](https://user-images.githubusercontent.com/24354709/46472745-d6813e80-c7e6-11e8-9d77-7d04ce30fbe6.png)

**Comment by logogriph2** [comment]
Отображение всё-таки есть, но показан только один источник. Другие не отображаются.
![systemtest_mce_source](https://user-images.githubusercontent.com/24354709/46472991-89519c80-c7e7-11e8-9bba-54bfc2a675a4.png)

**Comment by logogriph2** [comment]
Проблема была в высоком значении upper limit

**Comment by logogriph2** [comment]
А должно ли быть отображение сигнала в правом сегменте интерфейса?

**Comment by dmalt** [comment]
Да, должно, но оно обычно появляется не сразу, а через несколько секунд после старта.

**Comment by dmalt** [comment]
> Проблема была в высоком значении upper limit

Я не совсем понял, работает или нет. Для MCE вы должны наблюдать россыпь меняющих друг друга точек

**Comment by logogriph2** [comment]
Отображение источников работает. Отображение сигнала ЭЭГ - нет

**Comment by dmalt** [comment]
Только для MCE или для остальных методов тоже?

**Comment by dmalt** [comment]
Еще такой вопрос. Ползунок бежит, но сигналы плоские или отображение сигналов вообще стоит?

**Comment by logogriph2** [comment]
ползунок не бежит.
![mce_eeg_stop](https://user-images.githubusercontent.com/24354709/46613573-3e53c400-cb1c-11e8-8dc9-d92039c3dda9.jpg)

**Comment by logogriph2** [comment]
для бимформера отображения сигнала ээг тоже не происходит

**Comment by dmalt** [comment]
И никакого сообщения об ошибке?
Я все запускаю под linux, и у меня отображение сигналов работает. Попробую развернуть под windows, посмотрим, удастся ли воспроизвести такое поведение.

**Comment by dmalt** [comment]
Еще просьба. Пришлите пожалуйста cogingraph.log. Он должен лежать в корневой папке проекта

**Comment by logogriph2** [comment]
[cognigraph.log](https://github.com/dmalt/cognigraph/files/2456728/cognigraph.log)

**Comment by logogriph2** [comment]
при запуске через scripts/launch_test.py сигнал отображается. при systemtest - нет

**Comment by dmalt** [comment]
Спасибо! Это важная новость
